module/Swissbib/src/Swissbib/Util/Config/FlatArrayConverter.php

```diff
@@ -0,0 +1,145 @@
+<?php
```

Comment:
Do we really need so much code to read the config? 
Will fail php-cs-fixer-tests.

Reply:
The FlatArrayConverter class was intended to not only be used for this one purposes. My idea was to have a reusable utility component. If you are not happy with the solution, I can reduce it to a one ore two utility methods but I think it makes it less structured and less readable. Maybe the comments on the private methods could be removed at all, to reduce the noise and even the expressive comments could be shortened a little.

What is that php-cs-fixer-tests thing and how do I run it to see what fails?